python/metatensor-learn/README.rst

```diff
@@ -0,0 +1,4 @@
+metatensor-learn
+=====================
+
+This package contains building blocks for the atomistic machine learning models based on PyTorch and NumPy.
```

Comment:
This will need to be extended a bit, since it is rendered on PyPI. It's fine as is for now!